jdl/converters/jdl-to-json/jdl-to-json-relationship-converter.js

```diff
@@ -37,17 +37,19 @@ module.exports = {
  * Converts passed JDL relationships to JSON content.
  * @param {Array<JDLRelationship>} jdlRelationships - the relationships to convert.
  * @param {Array<String>} entityNames - all the entities' names.
+ * @param {Object} convertOptions - Convert options.
+ * @param {Boolean} [convertOptions.unidirectionalRelationships] - Whether to generate unidirectional relationships
  * @return {Map<String, Array<Object>>} a map having for keys entity names and for values arrays of JSON relationships.
  */
-function convert(jdlRelationships = [], entityNames = []) {
+function convert(jdlRelationships = [], entityNames = [], convertOptions = {}) {
```

Comment:
```suggestion
function convert(jdlRelationships = [], entityNames = [], conversionOptions = {}) {
```

convertOptions looks like a function (because it's an action), conversionOptions is a name which better indicates regular params/vars

Reply:
Great, done.